backend/src/main/java/capstone/storage/backend/storagebin/StorageBin.java

```diff
@@ -0,0 +1,6 @@
+package capstone.storage.backend.storagebin;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public record StorageBin(@JsonProperty("id") String locationNumber, String itemNumber, String amount) {
```

Comment:
Betrifft alle Records:
Ich empfehle eine Eigenschaft pro Zeile, damit man mit GIT die Änderungen besser nachvollziehen kann. Insbesondere natürlich für Records, die viele Eigenschaften haben (oder haben werden).
```suggestion
public record StorageBin(
    @JsonProperty("id") String locationNumber,
    String itemNumber,
    String amount) {
```